fabtests/common/shared.c

```diff
@@ -3818,6 +3819,9 @@ static int ft_pin_core(const char *core_list)
 		return EXIT_FAILURE;
 
 	return sched_setaffinity(0, sizeof(mask), &mask);
+#else
+	return EXIT_FAILURE;
+#endif
```

Comment:
For this, please define sched_setaffinity() in include/osx/osd.h.  We provide an alternative implementation of this function for Windows.  You may not need the configure changes then.

Reply:
I copied the implementation of `ofi_set_thread_affinity` into a newly added `common/unix/osd.c` in hopes it will work for both macOS and FreeBSD.

Reply:
Seems I flubbed this part. I'll have to come back to it later.